src/main.rs

```diff
@@ -450,39 +605,122 @@ fn get_setup(args: &[String]) -> Setup {
         exit(0);
     }
 
-    let verbose = matches.opt_present(VERBOSE);
-    setup_logging(verbose);
+    setup_logging(matches.opt_present(QUIET), matches.opt_present(VERBOSE));
```

Comment:
Quiet and verbose should be mutually exclusive (not allowed on the command line).

Reply:
Can do.

Reply:
I will make a note to the user with a warning. In my mind if both are given verbose should trump quiet.

Reply:
Done.